.github/workflows/Nightly_Perf_Env_CI.yml

```diff
@@ -6,8 +6,8 @@ name: Deploy Perf Env Nightly CI
 on:
  # Run on Nightly
 #  schedule:
-#    - cron: '0 4 * * 1-6' # run Monday through Saturday at 4 AM UTC/ 0 AM EDT
- # Run on weekly after IPI and operator completed
+#    - cron: '0 4 * * 0,2-6' # run Monday through Saturday at 4 AM UTC/ 0 AM EDT
```

Comment:
Update the comment to reflect Tuseday-Sunday.

Reply:
Fixed